features/admins/admin_email_blacklist.feature

```diff
@@ -59,3 +59,30 @@ Scenario: Banning a user's email should not affect their ability to post comment
     And I post the work "New Work"
     And I post a comment "here's a great comment"
   Then I should see "Comment created!"
+
+Scenario: Banning a user's email should prevent requesting invites
+  Given I am logged in as a "policy_and_abuse" admin
+  When I have banned the address "foo@bar.com"
+    And I log out
+    And account creation requires an invitation
+    And the invitation queue is enabled
+  When I am on the homepage
+    And I follow "Get an Invitation"
+  When I fill in "invite_request_email" with "foo@bar.com"
```

Comment:
Could you make this into an `And` step since there is directly after another `When`? Also, we prefer to use the user-facing field names ("Email" in this case) over the input names (invite_request_email). Same thing for the other test